src/hotspot/share/compiler/disassembler.cpp

```diff
@@ -637,15 +644,15 @@ void decode_env::print_insn_labels() {
     //---<  Block comments for nmethod  >---
     // Outputs a bol() before and a cr() after, but only if a comment is printed.
     // Prints nmethod_section_label as well.
-    if (_nm != NULL) {
+    if (_nm != nullptr) {
       _nm->print_block_comment(st, p);
     }
-    if (_codeBlob != NULL) {
+    else if (_codeBlob != nullptr) {
```

Comment:
This isn't code that you have changed - but still - why is _nm and _codeblob separate? print_block_comment is virtual and nmethod is a codeblob.